### DISCUSSION TITLE
Submit Case Fail due to missing Input data

### DISCUSSION THREAD

**Discussion by rytam2** [E3SM-Project/E3SM#5847] [E3SM model help]
Hello all, 

I am trying to run a **F2010** compset with 1 degree resolution (**ne30pg2_EC30to60E2r2**) on a local machine. 

The case successfully set up and built. But when I got to the point where the case is submitted, the case failed to build due to not finding all input data on any server. There is a long list of missing input files, so I only copied a couple in here. 

`...
File not found: atm2rof_fmapname = /data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/cpl/gridmaps/ne30pg2/map_ne30pg2_to_r05_mono.200220.nc, will attempt to download in check_input_data phase
File not found: atm2rof_smapname = /data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/cpl/gridmaps/ne30pg2/map_ne30pg2_to_r05_mono.200220.nc, will attempt to download in check_input_data phase
File not found: rof2ocn_liq_rmapname = /data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/cpl/cpl6/map_r05_to_EC30to60E2r2_smoothed.r150e300.201005.nc, will attempt to download in check_input_data phase
File not found: rof2ocn_ice_rmapname = /data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/cpl/cpl6/map_r05_to_EC30to60E2r2_smoothed.r150e300.201005.nc, will attempt to download in check_input_data phase
Checking that inputdata is available as part of case submission
Loading input file list: 'Buildconf/mpassi.input_data_list'
Model mpassi missing file mesh = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/EC30to60E2r2/seaice.EC30to60E2r2.210210.nc'
Model mpassi missing file snow_data = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/general/snicar_drdt_bst_fit_60_c04262019.nc'
Model mpassi missing file snicar_data = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/general/snicar_optics_5bnd_snow_and_aerosols.nc'
Model mpassi missing file graph48 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/EC30to60E2r2/mpas-seaice.graph.info.200908.part.48'
Loading input file list: 'Buildconf/eam.input_data_list'
Model eam missing file chlorine_loading_file = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart/ub/Linoz_Chlorine_Loading_CMIP6_0003-2017_c20171114.nc'
Model eam missing file drydep_srf_file = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mam/atmsrf_ne30pg2_200129.nc'
Model eam missing file linoz_data_file = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart/ub/linoz1850-2015_2010JPL_CMIP6_10deg_58km_c20171109.nc'
Model eam missing file tracer_cnst_file = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/oxid/oxid_1.9x2.5_L26_1850-2015_c20181106.nc'
Model eam missing file solar_data_file = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/solar/Solar_2010control_input4MIPS_c20181017.nc'
Model eam missing file prescribed_volcaero_file = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/volc/CMIP_DOE-ACME_radiation_average_1850-2014_v3_c20171204.nc'
Model eam missing file ncdata = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/inic/homme/20180716.DECKv1b_A3.ne30_oEC.edison.cam.i.2010-01-01-00000.nc'
Model eam missing file bnd_topo = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/topo/USGS-gtopo30_ne30np4pg2_16xdel2.c20200108.nc'
Model eam missing file mam_mom_filename = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mam/marine_BGC//monthly_macromolecules_0.1deg_bilinear_latlon_year01_merge_date.nc'
Model eam missing file ext_frc_specifier for SO2 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so2_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for SOAG = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_soag_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for bc_a4 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_bc_a4_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for num_a1 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a1_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for num_a2 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a2_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for num_a4 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a4_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for pom_a4 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_pom_a4_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for so4_a1 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a1_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file ext_frc_specifier for so4_a2 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a2_elev_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for DMS = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DMSflux.2010.1deg_latlon_conserv.POPmonthlyClimFromACES4BGC_c20190220.nc'
Model eam missing file srf_emis_specifier for SO2 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so2_surf_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for bc_a4 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_bc_a4_surf_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for num_a1 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a1_surf_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for num_a2 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a2_surf_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for num_a4 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a4_surf_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for pom_a4 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_pom_a4_surf_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for so4_a1 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a1_surf_1x1_2010_clim_c20190821.nc'
Model eam missing file srf_emis_specifier for so4_a2 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a2_surf_1x1_2010_clim_c20190821.nc'

... 

Loading input file list: 'Buildconf/mpassi.input_data_list'
Model mpassi missing file mesh = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/EC30to60E2r2/seaice.EC30to60E2r2.210210.nc'
Trying to download file: 'ice/mpas-cice/EC30to60E2r2/seaice.EC30to60E2r2.210210.nc' to path '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/EC30to60E2r2/seaice.EC30to60E2r2.210210.nc' using SVN protocol.
svn export failed with output:  and errput svn: E170000: URL 'https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/ice/mpas-cice/EC30to60E2r2/seaice.EC30to60E2r2.210210.nc' doesn't exist

Model mpassi missing file snow_data = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/general/snicar_drdt_bst_fit_60_c04262019.nc'
Trying to download file: 'ice/mpas-cice/general/snicar_drdt_bst_fit_60_c04262019.nc' to path '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/general/snicar_drdt_bst_fit_60_c04262019.nc' using SVN protocol.
svn export failed with output:  and errput svn: E170000: URL 'https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/ice/mpas-cice/general/snicar_drdt_bst_fit_60_c04262019.nc' doesn't exist

Model mpassi missing file snicar_data = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/general/snicar_optics_5bnd_snow_and_aerosols.nc'
Trying to download file: 'ice/mpas-cice/general/snicar_optics_5bnd_snow_and_aerosols.nc' to path '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/general/snicar_optics_5bnd_snow_and_aerosols.nc' using SVN protocol.
svn export failed with output:  and errput svn: E170000: URL 'https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/ice/mpas-cice/general/snicar_optics_5bnd_snow_and_aerosols.nc' doesn't exist

Model mpassi missing file graph48 = '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/EC30to60E2r2/mpas-seaice.graph.info.200908.part.48'
Trying to download file: 'ice/mpas-cice/EC30to60E2r2/mpas-seaice.graph.info.200908.part.48' to path '/data/keeling/a/rytam2/a/E3SM/E3SM-v2.0/E3SM_input_data/ice/mpas-cice/EC30to60E2r2/mpas-seaice.graph.info.200908.part.48' using SVN protocol.
svn export failed with output:  and errput svn: E170000: URL 'https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/ice/mpas-cice/EC30to60E2r2/mpas-seaice.graph.info.200908.part.48' doesn't exist

...
Checking server None with protocol None
Client protocol None not enabled
ERROR: Could not find all inputdata on any server
`

I checked the links from the error message, and indeed didn't find the files that are mentioned. Is there a way to go around this missing input file issue? If not, may I know where I can access these missing files? And is there a way to automatically get all missing files to download in the local machine? 

Would appreciate any help on this, and thank you so much in advance!

**Comment by mt5555** [comment]
Is this error as of today?  in which case it is probably that the E3SM inputdata server is down - scheduled maintenance starting Friday, through the weekend.  

The script above will check both the e3sm inputdata server, and then the ucar.edu inputdata server.  The files that are missing are very E3SM specific, so they will only be on the e3sm server.  As that is a common compset, they might also be available from Perlmutter at NERSC.

**Reply by rytam2** [reply]
I only tried downloading on Friday, but will try again on Monday. Thank you for the prompt reply!

**Comment by rytam2** [comment]
I tried building again and still got the same error today, which suggests the problem may not be related to the maintenance over the weekend. Is there a way I can directly download the input data from the e3sm server to my local machine? (Would downloading from Perlmutter at NERSC require an active NERSC account? If so, I don't have an account and this might not work.)

**Reply by rljacob** [reply]
You don't.  Try https://portal.nersc.gov/project/e3sm/inputdata

**Comment by maltrud** [comment]
I think the E3SM server is still down.  there was a full power outage starting friday and it looks like they haven't finished the project yet.

**Reply by rljacob** [reply]
The server is back up.

**Reply by rytam2** [reply]
The files are now downloaded!! Thank you so much!!

**Comment by rytam2** [comment]
Issue resolved

**Comment by rytam2** [comment]
Issue is resolved: missing input data failed to download due to E3SM server down due to scheduled maintenance.
